### ISSUE TITLE
Votes shows up as 0.00

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#336]
```
What steps will reproduce the problem?
1. Sync AniDB Votes
2. Vote in JMM Desktop and then refresh collection

What is the expected output? What do you see instead?

When voting in JMM Desktop or syncing my votes from anidb, it allways show up 
as 0.00 in JMM Desktop regardless of what i vote on AniDB or in JMM Desktop. 
The "general" rating works fine though.


What version of the product are you using? On what operating system?

JMM Desktop 3.1.21.0
JMM Server  3.1.21.0
MySQL Database
Windows 7

Please provide any additional information below.


```

Original issue reported on code.google.com by `xAndroid...@gmail.com` on 14 Nov 2012 at 9:32

Attachments:
- [faultyvote.png](https://storage.googleapis.com/google-code-attachments/jmm/issue-336/comment-0/faultyvote.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `werndly...@gmail.com` on 14 Nov 2012 at 10:37
- Added labels: **Milestone-Release3.0**, **Product-JMM**

**Comment by GoogleCodeExporter** [comment]
```
Do you use a language other than English for windows?
```

Original comment by `werndly...@gmail.com` on 23 Nov 2012 at 12:34

**Comment by GoogleCodeExporter** [comment]
```
Yes i am using Swedish.
```

Original comment by `radiorag...@gmail.com` on 24 Nov 2012 at 12:27

**Comment by GoogleCodeExporter** [comment]
```
Oops.. Looks like i was logged in to some other account when i wrote that, 
comment 3 was me and im using Swedish as stated.
```

Original comment by `xAndroid...@gmail.com` on 3 Dec 2012 at 7:19

**Comment by GoogleCodeExporter** [comment]
```
Ok thanks.

There was a previous issue with languages other than english having this issue.
It looks like it has re-occurred
```

Original comment by `werndly...@gmail.com` on 3 Dec 2012 at 10:35

**Comment by GoogleCodeExporter** [comment]
```
I see, though i have to say that is sounds a bit odd that the votes are 
influenced by the users OS language.
```

Original comment by `xAndroid...@gmail.com` on 4 Dec 2012 at 5:21

**Comment by GoogleCodeExporter** [comment]
```
Yes, I think the problem is with decimals from memory.

English uses "." for decimals
Swedish/German uses "," 
```

Original comment by `werndly...@gmail.com` on 5 Dec 2012 at 1:42

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.24
```

Original comment by `werndly...@gmail.com` on 5 Dec 2012 at 2:02
- Changed state: **Fixed**
